src/libraries/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -677,28 +677,17 @@ public static unsafe string Title
 
         public static void Beep()
         {
-            const char BellCharacter = '\u0007'; // Windows doesn't use terminfo, so the codepoint is hardcoded.
-
             if (!Console.IsOutputRedirected)
             {
-                Console.Out.Write(BellCharacter);
-                return;
-            }
-
-            if (!Console.IsErrorRedirected)
-            {
-                Console.Error.Write(BellCharacter);
-                return;
+                ReadOnlySpan<byte> bell = "\u0007"u8; // Windows doesn't use terminfo, so the codepoint is hardcoded.
```

Comment:
Does this need to be 2-byte char when `Console.OutputEncoding.CodePage == UnicodeCodePage`?

Reply:
Great question! I've run following code:

```cs
using System.Text;

const int UnicodeCodePage = 1200;

foreach (Encoding encoding in new Encoding[] { Encoding.UTF8, Encoding.Unicode })
{
    Console.OutputEncoding = encoding;
    Console.WriteLine(Console.OutputEncoding);
    Console.Beep();
    Console.WriteLine(Console.OutputEncoding.CodePage == UnicodeCodePage);
    Thread.Sleep(TimeSpan.FromSeconds(1.0));
}
```

with corerun.exe and it works as expected, so I guess it does not need 2-byte char for Unicode. But my testing was limited to Win 11.

Reply:
Did you repro produced two jingles or just one? It only produces one jingle for me (from the UTF8 encoding iteration).

Try the following simpler repro - it does not produce any sound for me with this change:
```
using System.Text;

Console.OutputEncoding = Encoding.Unicode;
Console.Beep();
Thread.Sleep(10000);
```

Reply:
> Did you repro produce two jingles or just one?

Two (that is why I've added the sleep to the repro to have a pause between them).

> Try the following simpler repro 

It works on my machine. Does using a 2-byte buffer solves it for you?

Reply:
@jkotas please excuse me, I was testing wrong implementation, as running the following command does not replace `System.Console.dll` that is next to `corerun.exe`:

```cmd
.\dotnet.cmd build .\src\libraries\System.Console\tests\System.Console.Tests.csproj -c Release /t:Test
```

But running following does:

```cmd
.\dotnet.cmd build .\src\libraries\System.Console\System.Console.sln -c Release
```

I was able to repro and will send a fix very soon.

Reply:
Fix: https://github.com/dotnet/runtime/pull/105094